src/providers.ts

```diff
@@ -31,7 +33,31 @@ const LD_MODE: DocumentFilter = {
 	scheme: 'file',
 };
 
-export function register(ctx: ExtensionContext, config: Configuration, flagStore: FlagStore, api: LaunchDarklyAPI) {
+export async function register(ctx: ExtensionContext, config: Configuration, api: LaunchDarklyAPI) {
+	var flagStore;
+
+	try {
+		const flags = await api.getFeatureFlags(config.project, config.env);
+		const arrayToObject = (array: Array<FeatureFlag>) =>
```

Comment:
Looks like we've declared this method in multiple places. Since it gets a little funky with typescript, maybe we could use lodash's [`keyBy`](https://lodash.com/docs/4.17.15#keyBy): 

```
keyBy(flags, 'key')
```

Reply:
that's awesome! Worked like a charm.